diff --git a/dist/esm/utils.js b/dist/esm/utils.js
index bfbf0c126283dc854c5f353145756d891f0e38db..6520e44481d412b80cc44c598bd689f8a78b9e34 100644
--- a/dist/esm/utils.js
+++ b/dist/esm/utils.js
@@ -4,7 +4,7 @@ import * as React$1 from "react";
 * React.use if available (React 19+), undefined otherwise.
 * Use dynamic lookup to avoid Webpack compilation errors with React 18.
 */
-var reactUse = React$1["use"];
+var reactUse = Reflect.get(React$1, "use");
 var useLayoutEffect = typeof window !== "undefined" ? React$1.useLayoutEffect : React$1.useEffect;
 /**
 * Taken from https://www.developerway.com/posts/implementing-advanced-use-previous-hook#part3
